src/Mesh/DerivedData.C

```diff
@@ -1099,7 +1183,89 @@ genInpofaTet( std::size_t ntfac,
     }
   }
 
+  // this fills the boundary face-node connectivity part
+  // consistent with triinpoel
+  for (std::size_t f=0; f<nbfac; ++f)
+  {
+    icoun = nnpf * f;
+    for(std::size_t i=0; i<nnpf ; ++i)
+    {
+      inpofa[icoun+i] = triinpoel[icoun+i];
+    }
+  }
+  }
+
   return inpofa;
 }
+        
+std::vector< std::size_t >
+genBelemTet( std::size_t nbfac,
+              const std::vector< std::size_t >& inpofa,
+              const std::pair< std::vector< std::size_t >,
+                               std::vector< std::size_t > >& esup )
+// *****************************************************************************
+//  Generate derived data, host elements for boundary faces
```

Comment:
"Host" elements are? The domain element (tetrahedron) sharing a side/face with a boundary? If so, let's make it more clear for those who do not speak this language that well (yet).